content/en/docs/tasks/configure-pod-container/resize-container-resources.md

```diff
@@ -2,264 +2,192 @@
+
+If a node has pods with a pending or incomplete resize (see [Pod Resize Status](#pod-resize-status) below), the scheduler uses the *maximum* of a container's desired requests, allocated requests, and actual requests from the status when making scheduling decisions.
 
 ## {{% heading "prerequisites" %}}
 
 
 {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
-The `InPlacePodVerticalScaling` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) must be enabled
-for your control plane and for all nodes in your cluster.
+The `kubectl` client version must be at least v1.32 to use the `--subresource=resize` flag.
 
-## Container Resize Policies
+## Pod Resize Status
 
-Resize policies allow for a more fine-grained control over how pod's containers
-are resized for CPU and memory resources. For example, the container's
-application may be able to handle CPU resources resized without being restarted,
-but resizing memory may require that the application hence the containers be restarted.
+The Kubelet updates the Pod's status conditions to indicate the state of a resize request:
 
-To enable this, the Container specification allows users to specify a `resizePolicy`.
-The following restart policies can be specified for resizing CPU and memory:
-* `NotRequired`: Resize the container's resources while it is running.
-* `RestartContainer`: Restart the container and apply new resources upon restart.
+* `type: PodResizePending`: The Kubelet cannot immediately grant the request. The `message` field provides an explanation of why.
+    * `reason: Infeasible`: The requested resize is impossible on the current node (for example, requesting more resources than the node has).
+    * `reason: Deferred`: The requested resize is currently not possible, but might become feasible later (for example if another pod is removed). The Kubelet will retry the resize.
+    
 
-If `resizePolicy[*].restartPolicy` is not specified, it defaults to `NotRequired`.
+* `type: PodResizeInProgress`: The Kubelet has accepted the resize and allocated resources, but the changes are still being applied. This is usually brief but might take longer depending on the resource type and runtime behavior. Any errors during actuation are reported in the `message` field (along with `reason: Error`).
 
-{{< note >}}
-If the Pod's `restartPolicy` is `Never`, container's resize restart policy must be
-set to `NotRequired` for all Containers in the Pod.
-{{< /note >}}
+## Container Resize Policies
 
-Below example shows a Pod whose Container's CPU can be resized without restart, but
-resizing memory requires the container to be restarted.
+You can control whether a container should be restarted when resizing by setting `resizePolicy` within the container specification. This allows fine-grained control based on resource type (CPU or memory).
```

Comment:
What about extended resources (eg from [device plugins](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/))? The docs should make the story clear for those.

Reply:
This is covered under key-concepts (only CPU & Memory are mutable) and limitations. Do you have any suggestions of how to raise the visibility of this?